src/index.js

```diff
@@ -237,6 +240,7 @@ const forms = plugin.withOptions(function (options = { strategy: 'base' }) {
           padding: '0',
           'font-size': 'unset',
           'line-height': 'inherit',
+          '--tw-shadow': '0 0 #0000',
```

Comment:
Is this one needed? I can't think for sure but maybe it's not since we don't use `box-shadow` for any focus styles on file inputs?

Reply:
Aha, yeah I added it because I wanted to make sure that I got all the `base` cases, but we are indeed not adding any focus styles for this one. 
However if we eventually do, probably not though, then we can't forget to add these back.